lib/active_record/connection_adapters/cockroachdb_adapter.rb

```diff
@@ -155,20 +154,6 @@ def max_transaction_retries
         @max_transaction_retries ||= @config.fetch(:max_transaction_retries, 3)
       end
 
-      # CockroachDB 20.1 can run queries that work against PostgreSQL 10+.
-      def postgresql_version
-        100000
-      end
```

Comment:
This bit I'm not sure about, let's see what the tests are saying about it

Reply:
Looked around, this is a bit of an issue because of some of the `support...?` method in the pg adapter are depending on this for checks, and the crdb version is way higher than pg, making every check true. But it is just a matter of copying those checks and verifying if they are crdb compliant. Overall, I think we're better as we are now